scripts/gitlab/package-snap.sh

```diff
@@ -3,7 +3,7 @@
 set -e # fail on any error
 set -u # treat unset variables as error
 case ${CI_COMMIT_REF_NAME} in
-  master|*v2.1*|gitlab-next) export GRADE="devel";;
+  nightly|*v2.1*) export GRADE="devel";;
```

Comment:
Are we going to publish nightlies only from the special `nightly` branch? Why not from the `master`?

Reply:
@5chdn Alternatively, we can avoid triggering the actual build-and-package on `master` on every push (only tests), and trigger the builds only once per day with a cron-like trigger (Gitlab CI already supports those). This way, there would be no need in maintaining a separate `nightly` branch manually.

If you agree with me that not having to maintain separate `nightly` is desired, please open a separate issue for this (or let me know, and I'll do this).

Reply:
we are doing that. nightly is a tag.